modules/apps/accessibility/accessibility-menu-web/src/main/resources/META-INF/resources/js/AccessibilitySetting.tsx

```diff
@@ -35,6 +41,12 @@ const AccessibilitySetting = ({
 			onToggle={onChange}
 			toggled={value}
 		/>
+
+		<span className="toggle-switch-text">
+			<Text color="secondary" id={`accessibility-help-${index}`} size={3}>
+				{description}
+			</Text>
+		</span>
```

Comment:
I think the ideal for this here would be `Form.Group` together with `Form.FeedbackGroup`, for example:

```jsx
<li>
  <ClayForm.Group>
    <ClayToggle
      aria-describedby={`accessibility-help-${index}`}
      disabled={disabled}
      label={label}
      onToggle={onChange}
      toggled={value}
    />

    <ClayForm.FeedbackGroup>
      <Text color="secondary" id={`accessibility-help-${index}`} size={3}>
				{description}
			</Text>
    </ClayForm.FeedbackGroup>
  </ClayForm.Group>
</li>
```

I think `li` wouldn't need `mb-3` anymore because the FormGroup takes care of adding a bottom margin, I'm not sure anymore, but we have to check.

Reply:
I'll check it and get back to you.

Reply:
Fixed, no need to add `mb-3`.